### ISSUE TITLE
Editing the position of a gravity affector also edits the position of the emitter.

### ISSUE THREAD

**Issue by rkettering** [anura-engine/anura#168]
There seem to, in general, be some issues with similarly named fields editing multiple fields at once.

**Comment by sweetkristas** [comment]
Yeah I've addressed a few instances of these previously. Just need to add
some ImGui::PushID(...); ImGui::PopID(); calls in the appropriate places.

ImGui treats fields with the same label as the same for control logic
purposes.

**Comment by rkettering** [comment]
I'll keep you posted with any further ones I find.
